Tools/CISupport/ews-build/steps.py

```diff
@@ -1710,10 +1709,18 @@ def start(self):
         return None
 
     def getResultSummary(self):
+        if self.results == SKIPPED:
```

Comment:
Better to not have this handling of SKIPPED case as I explained in https://github.com/WebKit/WebKit/pull/231#discussion_r839863895